apps/confidential/confidential.go

```diff
@@ -799,3 +818,72 @@ func (cca Client) Account(ctx context.Context, accountID string) (Account, error
 func (cca Client) RemoveAccount(ctx context.Context, account Account) error {
 	return cca.base.RemoveAccount(ctx, account)
 }
+
+// WithFMIPath specifies the path to a federated managed identity.
+// The path should point to a valid FMI configuration file that contains the necessary
+// identity information for authentication.
+//
+// Example:
+//
+//	result, err := client.AcquireTokenByCredential(ctx, scopes, confidential.WithFMIPath("fmi/path"))
```

Comment:
suggest removing this example because the compiler won't check it and it's trivial (the hard thing here is knowing when to use this option and how to get a valid FMI configuration file, not calling the function)